apps/web/package.json

```diff
@@ -11,31 +11,63 @@
     "test:watch": "cross-env DEV=true yarn test --watch"
   },
   "dependencies": {
+    "@chakra-ui/anatomy": "2.2.2",
+    "@chakra-ui/cli": "^2.4.1",
```

Comment:
that's definitely a dev dependency; please also adjust others too
in the desktop app, all the deps are dev deps because of how electron-builder works